docs/source/cli/xcube_patch.rst

```diff
@@ -53,6 +57,10 @@ value of the attribute ``long_name`` of variable ``conc_chl``:
 
     zarr_consolidated_format: 1
     metadata:
+
+      .zattrs:
+        TileSize: __delete__
```

Comment:
Please mention in the description of the example what this will do.

Reply:
It is described. Please read carefully.

Reply:
I am talking about lines 52-54. You should state there that this configuration will also remove an attribute.